crates/cosmos/cosmos-integration-tests/src/contexts/bootstrap_legacy.rs

```diff
@@ -94,6 +101,8 @@ delegate_components! {
             ChainStoreDirGetterComponent,
             ChainCommandPathGetterComponent,
             AccountPrefixGetterComponent,
+            DynamicGasGetterComponent,
```

Comment:
We probably want to always disable dynamic gas when bootstrapping legacy chain. We can define a `ReturnNoDynamicGas` provider that always return `None`, and wire with that provider here.

Reply:
Done in 15a2b78133a23e13b4678a78ff0d0926fdf0bdf2